.travis.yml

```diff
@@ -1,13 +1,23 @@
 sudo: false
 language: scala
+services:
+- redis-server
 scala:
 - 2.11
 jdk:
 - oraclejdk8
 env:
   global:
-  - BRIDGE_USER=travis
+  - BRIDGE_USER=heroku
   - BRIDGE_ENV=dev
+  - WEBSERVICES_URL="https://webservices-develop.sagebridge.org"
+  - HOST_POSTFIX="-develop.sagebridge.org"
+  - UDD_SQS_QUEUE_URL="https://sqs.us-east-1.amazonaws.com/649232250620/Bridge-WorkerPlatform-Request-dev"
+  - UPLOAD_CMS_CERT_BUCKET="org-sagebridge-upload-cms-cert-develop"
+  - CONSENTS_BUCKET="org-sagebridge-consents-develop"
+  - UPLOAD_BUCKET="org-sagebridge-upload-develop"
+  - UPLOAD_CMS_PRIV_BUCKET="org-sagebridge-upload-cms-priv-develop"
+  - ATTACHMENT_BUCKET="org-sagebridge-attachment-develop"
   - secure: SsM07kjCSWjGTFDhkCtTuZdM5WrChFXgRmeMMRJr4JlbtJoaaVUXcXDofZ+4NrESiOvW7CiRXK6Oz3I9eL7vn8RCwlN5jf2OlmD4vKIiiINlvuuEe6hEu738RO7nUGN8SFh2k4ryzJ6XldghjBiBvBEqzVhwXteLhF3AVCsRB1s=
```

Comment:
We don't need any of those. Configs in .travis.yml are only needed for build and test. Most of them come from bridge-server.conf in the Github repo, so they don't need to be replicated in the .travis.yml. The rest are AWS keys, which we prefer to specify in the Travis build directly, so that they are easier to change when needed.